projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -1319,6 +1319,9 @@ public function export_to_gdrive() {
 		require_once JETPACK__PLUGIN_DIR . '_inc/lib/class-jetpack-google-drive-helper.php';
 		$sheet = Jetpack_Google_Drive_Helper::create_sheet( $user_id, $spreadsheet_title, $sheet_data );
 
+		$tracking = new Automattic\Jetpack\Tracking();
```

Comment:
While this is NOT a namespaced file, we could also use the syntax mentioned above, just for the sake of consistency.